components/reddit/sources/new-comments-on-a-subreddit/new-comments-on-a-subreddit.js

```diff
@@ -0,0 +1,125 @@
+const common = require("../common");
+const { reddit } = common.props;
+
+module.exports = {
+  ...common,
+  key: "new-comments-on-a-subreddit",
```

Comment:
Just to confirm, since you're passing a Post ID, won't this only watch a specific post within a subreddit for comments? Should we rename the source / description accordingly?

Reply:
Well, the thing is in Reddit lingo a subreddit can be a post or an online community. We know which connotation subreddit refers to by the context. This is clarified in the prop PostID description,

"Pipedream will emit a new event when a comment is posted to the subreddit pointed by this ID36."

On the other hand, in Reddit API a developer would like for /r/subreddit/comments
So I think it is intuitive that the user would look for the related source in Pipedream with the subreddit in the source name, and then cross-reference that `Post ID` in Pipedream source is `article` in Reddit API docs.

Reply:
Ok, as per the Slack conversation, the source was renamed to `New comments on a post`